lib/units/websocket/index.js

```diff
@@ -334,6 +336,38 @@ module.exports = function(options) {
         .on('user.settings.reset', function() {
           dbapi.resetUserSettings(user.email)
         })
+        .on('user.keys.accessToken.generate', function(data) {
+          var expiry = Date.now() + 100 * 365 * 24 * 3600
```

Comment:
I think we should patch jwtutil instead, since adding clearly unnecessary data is a waste. Currently it adds a default expiry if there's none given, so I understand your choice here. However I would like to either

1. Remove the default expiry entirely, and simply ensure that it's being added where it's needed.
2. or, if `expiry === null`, then don't add the exp option at all. It would still set the default expiry if expiry isn't given at all (it's `undefined`).

I think I'd personally prefer option 1. If you can implement it in this PR, that would be great.

This would also make the token a bit smaller.